sormas-backend/src/main/java/de/symeda/sormas/backend/common/ChangeDateFilterBuilder.java

```diff
@@ -61,10 +73,23 @@ public Predicate build() {
 			parent = parent.join(joinField, JoinType.LEFT);
 		}
 
+		Predicate filter;
 		if (dateExpression == null) {
```

Comment:
Same as in BaseAdoService createChangeDateFilter.

### First scenario: no uuid passed
change date has to be greater than change date (lower, so as it currently is)

### Second scenario: uuid passed
Behave as described in issue

Reply:
Date comparisons in the `CriteriaBuilder` have precision milliseconds, see above.
Edge cases are modeled in `ImmunizationFacadeEjbTest.testGetAllSince`.

Reply:
Refactored.